src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs

```diff
@@ -156,7 +156,11 @@ public PackageBuilder GetPackageBuilder(IPackTaskRequest<IMSBuildItem> request)
             }
             if (!string.IsNullOrEmpty(request.RepositoryUrl) || !string.IsNullOrEmpty(request.RepositoryType))
```

Comment:
should we revisit this condition?

Reply:
do you think the rest of the info could be used without this?